hack/generated/pkg/genruntime/resolved_references.go

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package genruntime
+
+// ResolvedReferences is a set of references which have been resolved for a particular resource.
+// The special self field is the fully qualified ARM ID of the resource that this ResolvedReferences applies to.
+type ResolvedReferences struct {
```

Comment:
I don't get why you're bundling these two things together into one struct. Why is name in here?

Reply:
Pulled it back out

Reply:
Removed name from this